shared-module/displayio/__init__.c

```diff
@@ -108,10 +111,16 @@ void displayio_background(void) {
 
 }
 
-void common_hal_displayio_release_displays(void) {
+static void common_hal_displayio_release_displays_impl(bool keep_primary) {
     // Release displays before busses so that they can send any final commands to turn the display
     // off properly.
+    if (!keep_primary) {
+        primary_display_number = -1;
+    }
     for (uint8_t i = 0; i < CIRCUITPY_DISPLAY_LIMIT; i++) {
+        if (i == primary_display_number) {
```

Comment:
I think this is what you want.

```suggestion
        if (keep_primary && i == primary_display_number) {
```

Reply:
That's originally what I had, but I changed it. Here's my logic: If `!keep_primary`, then `primary_display_number` was updated just above and will always be `-1` here. -1 will never be equal to the loop counter.